src/login/loginctl.c

```diff
@@ -240,7 +240,7 @@ static int list_sessions(int argc, char *argv[], void *userdata) {
                         return table_log_add_error(r);
 
                 if (i.idle_hint)
-                        r = table_add_cell(table, NULL, TABLE_TIMESTAMP_RELATIVE, &i.idle_hint_timestamp.monotonic);
+                        r = table_add_cell(table, NULL, TABLE_TIMESTAMP_RELATIVE, &i.idle_hint_timestamp);
```

Comment:
maybe we should add TABLE_TIEMSTAMP_RELATIVE_MONOTONIC?

i mean, generally monotonic time for things like this are really preferable I'd say

Reply:
-> #27786